src/Entity/EventCategory.php

```diff
@@ -22,4 +22,19 @@
  */
 class EventCategory extends BaseEventCategory
 {
+    /**
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\EventGroupCategory", inversedBy="eventCategories")
+     * @ORM\JoinColumn(name="event_group_category_id", referencedColumnName="id", nullable=false)
+     */
```

Comment:
1) what about validation (`Assert`)?
2) why do you set `name` and `refColName` your self?

Reply:
Assert\NotBlank Added, naming removed.